src/diagnostics/internal/drawdata/drawdatagenerator.h

```diff
@@ -27,13 +27,17 @@
 #include "draw/types/drawdata.h"
 #include "../../diagnosticstypes.h"
 
+#include "modularity/ioc.h"
+#include "engraving/rendering/iscorerenderer.h"
+
 namespace mu::engraving {
 class MasterScore;
 }
 
 namespace mu::diagnostics {
 class DrawDataGenerator
 {
+    INJECT(engraving::rendering::IScoreRenderer, scoreRender)
```

Comment:
Now, some classes still use `render` in `INJECT`. But we can also fix that later, if you think that's better. For the rest, it looks good to me!